qa/python_models/execute_error/model.py

```diff
@@ -41,10 +41,9 @@ def execute(self, requests):
         """
         responses = []
         for request in requests:
-            input_tensors = request.inputs()
-            in_all = input_tensors[0].numpy_array()
```

Comment:
Can we call the function as_numpy() to be consistent with our client APIs?

Reply:
Fixed.